app/_src/kubernetes-ingress-controller/reference/feature-gates.md

```diff
@@ -91,3 +94,12 @@ private keys in `Certificate` entities and `Consumer` entities' credentials.
 > **Warning:** `KongPlugin`'s and `KongClusterPlugin`'s `config` fields are not sanitized. If you have sensitive information
 > in your `KongPlugin`'s `config` field, it will be sent to Konnect as is. To avoid that, please consider using
 > [KongVault](/kubernetes-ingress-controller/{{page.release}}/reference/custom-resources/#kongvault).
+
+### CombinedServicesFromDifferentHTTPRoutes
+
+The `CombinedServicesFromDifferentHTTPRoutes` feature enables translating `HTTPRoute` rules
+having the same set of backends (combination of namespace, name, port and weight) from different `HTTPRoute`s in the same namespace
```

Comment:
```suggestion
with the same set of backends
```

Reply:
backends belong to a rule in `HTTPRoute` but not the whole `HTTPRoute`. So I think the doc should mention that the consolidation is applied on rules of `HTTPRoute`s.